stable/kong/templates/psp.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: policy/v1beta1
```

Comment:
Why is this resource being always created? Please guard it with an `if` block.

Reply:
good catch